mlos_bench/mlos_bench/storage/sql_schema.py

```diff
@@ -0,0 +1,184 @@
+        # fixed for a particular trial config.
+        self.config_param = Table(
+            "config_param",
+            self._meta,
+            Column("config_id", Integer, nullable=False),
+            Column("param_id", String(255), nullable=False),
+            Column("param_value", String(255)),
+
+            PrimaryKeyConstraint("config_id", "param_id"),
+            ForeignKeyConstraint(["config_id"], [self.config.c.config_id]),
+        )
+
+        # Values of additional non-tunable parameters of the trial,
+        # e.g., scheduled execution time, VM name / location, number of repeats, etc.
+        self.trial_param = Table(
+            "trial_param",
+            self._meta,
+            Column("exp_id", String(255), nullable=False),
+            Column("trial_id", Integer, nullable=False),
+            Column("param_id", String(255), nullable=False),
+            Column("param_value", String(255)),
+
+            PrimaryKeyConstraint("exp_id", "trial_id", "param_id"),
+            ForeignKeyConstraint(["exp_id", "trial_id"],
+                                 [self.trial.c.exp_id, self.trial.c.trial_id]),
+        )
+
+        self.trial_result = Table(
+            "trial_result",
+            self._meta,
+            Column("exp_id", String(255), nullable=False),
+            Column("trial_id", Integer, nullable=False),
+            Column("metric_id", String(255), nullable=False),
+            Column("metric_value", String(255)),
+
+            PrimaryKeyConstraint("exp_id", "trial_id", "metric_id"),
+            ForeignKeyConstraint(["exp_id", "trial_id"],
+                                 [self.trial.c.exp_id, self.trial.c.trial_id]),
+        )
+
+        self.trial_telemetry = Table(
+            "trial_telemetry",
+            self._meta,
+            Column("exp_id", String(255), nullable=False),
+            Column("trial_id", Integer, nullable=False),
+            Column("ts", DateTime, nullable=False, default="now"),
+            Column("metric_id", String(255), nullable=False),
+            Column("metric_value", String(255)),
+
+            UniqueConstraint("exp_id", "trial_id", "ts", "metric_id"),
+            ForeignKeyConstraint(["exp_id", "trial_id"],
+                                 [self.trial.c.exp_id, self.trial.c.trial_id]),
+        )
+
+        _LOG.debug("Schema: %s", self._meta)
+
+    def create(self) -> 'DbSchema':
+        """
+        Create the DB schema.
+        """
+        _LOG.info("Create the DB schema")
+        self._meta.create_all(self._engine)
+        return self
+
+    def __repr__(self) -> str:
+        """
+        Produce a string with all SQL statements required to create the schema
+        from scratch in current SQL dialect.
+
+        That is, return a collection of CREATE TABLE statements and such.
+        NOTE: this method is quite heavy! We use it only once at startup
+        to log the schema, and if the logging level is set to DEBUG.
```

Comment:
```suggestion
        to log the schema, and if the logging level is set to DEBUG.
        
        This can also be done manually like so:
        python -e "from mlos_bench.storage.sql.schema import DbSchema; print(DbSchema);"
```

In fact, if we wanted, we could do something like the following to have the done automatically:

```python
if __name__ == "__main__":
    print(DbSchema)
```

And then all one needs to do is:

```sh
python mlos_bench/storage/sql/schema.py
```

Reply:
oh wait, no - we need to instantiate an SQLAlchemy DB engine, and that, in turn, requires us to load the JSONC stirage config etc. etc. 
There might be a way, but let's do it in some other PR

Reply:
Sounds good.  I guess I didn't look too closely, but I thought you were using some dummy engine for the schema printing, so it would have been fine.  Meh, not super important.  I can just imagine the need to debug that and easy access to the SQL text would be nice.